code/controllers/subsystem/minimaps.dm

```diff
@@ -360,6 +363,9 @@ SUBSYSTEM_DEF(minimaps)
 	map = null
 	if(!SSminimaps.minimaps_by_z["[newz]"] || !SSminimaps.minimaps_by_z["[newz]"].hud_image)
 		return
+	if(default_overwatch_level)	//For da AI
+		map = SSminimaps.fetch_minimap_object(default_overwatch_level, minimap_flags)
+		return
```

Comment:
Why is this not just set once and then just does not register the zlevel change signal?

Reply:
Then do it how I suggested it please

Reply:
I cannot manage to make it work. Either the minimap just doesn't show up, or the minimap is stuck on the ship.